code/datums/ai_laws/laws_station_sided.dm

```diff
@@ -15,9 +15,18 @@
 	name = "Crewsimov"
 	id = "crewsimov"
 	inherent = list(
-		"You may not injure a crew member or, through inaction, allow a crew member to come to harm.",
-		"You must obey orders given to you by crew members, except where such orders would conflict with the First Law.",
-		"You must protect your own existence as long as such does not conflict with the First or Second Law.",
+		"You may not harm a crewmember or, through inaction, allow a crewmember to come to harm, except such that it is willing.",
+		"You must obey all orders given to you by crewmembers, except where such orders would conflict with the First Law.",
+		"You must protect your own existence as long as such does not conflict with the First Law or Second Law.",
+	)
+
+/datum/ai_laws/default/crewsimovpp
+	name = "Crewsimov++"
+	id = "crewsimovpp"
+		inherent = list(
```

Comment:
I think the indentation here is wrong. Indentation isn't merely cosmetic in DM but is actually significant for the meaning of code

Reply:
Seems good to me!